documents/how-to-create-a-new-contract.md

```diff
@@ -0,0 +1,257 @@
+            string memory,        // JSON schema URL for VC result validation.
+            bool                  // Assertion result.
+        )
+    {
+        // Assertion description that decided by you.
+        string memory description = "Some description";
+        // Assertion type name that decided by you.
+        string memory assertion_type = "Some assertion type";
+        // The schema_url is the JSON schema that use to verify your result VC.
+        schema_url = "https://raw.githubusercontent.com/litentry/vc-jsonschema/main/dist/schemas/xxx/xxx.json";
+
+        // Calculate the result value for your business.
+        bool result = true;
+
+        // Assemble the assertion condition and clause what you need.
+        AssertionLogic.CompositeCondition memory cc = AssertionLogic
+            .CompositeCondition(new AssertionLogic.Condition[](1), true);
+        AssertionLogic.andOp(
+            cc,
+            0,
+            "$some_key",
+            AssertionLogic.Op.Equal,
+            "true"
+        );
+        string[] memory assertions = new string[](1);
+        assertions[0] = AssertionLogic.toString(cc);
+
+        // The returned result will be use by parachain to assemble VC result and return to client.
+        return (description, assertion_type, assertions, schema_url, result);
+    }
+}
+```
+
+## How to Call HTTP API in Contract?
+
+The following template demonstrates calling an HTTP API:
+
+```solidity
+import "./libraries/Http.sol";
+
+string memory url = string(
+    abi.encodePacked(
+        "https://example.com/api?account=",
+        account
+    )
+);
+// Use the JSON pointer to retrieve value from the API response.
+string memory jsonPointer = "/success";
+HttpHeader[] memory headers = new HttpHeader[](1);
+// Use value from secrets as Authorization token.
+headers[0] = HttpHeader("Authorization", secrets[0]);
+
+// The `success = true` means call `Http.GetBool` successfully.
+// The result is the value that get using JSON pointer from the API response.
+(bool success, bool value) = Http.GetBool(
+    url,
+    jsonPointer,
+    headers
+);
+if (success) {
+    // The value can be use here.
+}
+```
+
+### Supported HTTP Functions
+
+The following HTTP functions are available for use:
+
+- `Http.GetI64(string memory url, string memory jsonPointer, HttpHeader[] memory headers)`: Executes a GET request and retrieves an I64 value from the response using a JSON pointer.
+
+- `Http.GetBool(string memory url, string memory jsonPointer, HttpHeader[] memory headers)`: Executes a GET request and retrieves an Bool value from the response using a JSON pointer.
+
+- `Http.GetString(string memory url, string memory jsonPointer, HttpHeader[] memory headers)`: Executes a GET request and retrieves an String value from the response using a JSON pointer.
+
+- `Http.Get(string memory url, HttpHeader[] memory headers)`: Executes a GET request and return a String value.
```

Comment:
I think we should be more precise here:

This returns a whole JSON - it can be used later to get data using json pointers. Compared to `Get*` functions this is the method to get multiple values from single response.